### PR TITLE
Setup monitoring

### PR DESCRIPTION
**Pull request by oddg**
This PR includes setting up logdna to collect the logs and setting Sentry to
collect the errors.

### CODE REVIEW

**Comment by oddg** [comment]
I did a test deployment on the routificUrlShortener-testing environment and it worked (see https://github.com/routific/url-shortener-node-mongo-express/commit/c2b56c640f7f6cdeeda5e8099fb92c4e7e0b28f4).

After deploying, we still need to forward the application logs to `/var/log/url-shortener.log`. For that we change the AWS EB Node Command to `npm run production`

**Comment by oddg** [comment]
@rogertsui0923 Ready for review.

File: config.js [hunk 1]
```diff
@@ -1,14 +1,47 @@
...
+config.loggerConfig = {
+  name: 'url-shortener',
+  level: loadFromEnvironment('LOG_LEVEL', { defaultValue: 'info' }),
+  clsNamespace: 'fi.routi'
+}
+
+if (env === 'production') {
+  config.loggerConfig.sentryConfig = {
+    dsn: loadFromEnvironment('SENTRY_DSN', { fail: true }),
+    stack: 'production',
```
Comment (rogertsui0923) [inline config.js]: Do we not want a staging environment for the URL shortener?

Comment (oddg) [reply config.js]: I've been puzzled by this question but I think it's fine to skip it. We don't ship code on it very often. If we do start modifying it on a regular basis we can reassess.

File: .ebextensions/94_logdna.config [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      name=LogDNA_packages
+      baseurl=http://repo.logdna.com/el6/
+      enabled=1
+      gpgcheck=0
+  "/etc/logdna.conf" :
+    mode: "000644"
+    owner: root
+    group: root
+    content: |
+      key = `{"Fn::GetOptionSetting": {"Namespace": "aws:elasticbeanstalk:application:environment", "OptionName": "LOGDNA_INGESTION_KEY", "DefaultValue": ""}}`
```
Comment (rogertsui0923) [inline .ebextensions/94_logdna.config]: Cool.

**Pull request merged**